self_hosted/ast.jou

```diff
@@ -482,6 +486,14 @@ class AstIfStatement:
         free(self->if_and_elifs)
         self->else_body.free()
 
+class AstPass:
```

Comment:
We don't need this class. It's fine to not have a `union` member for all different kinds of `AstStatement`s. There's no `AstBreak` or `AstContinue` classes either.